ansible/roles/operator-pipeline/templates/openshift/tasks/github-pipelinerun-summary.yml

```diff
@@ -130,6 +139,30 @@ spec:
           DOC_LINK="https://github.com/redhat-openshift-ecosystem/certification-releases/blob/main/4.9/ga/troubleshooting.md"
         fi
 
+        if [[ "$(params.pipeline_name)" == "operator-release-pipeline" && ("$(params.pipeline_status)" == "Succeeded" || "$(params.pipeline_status)" == "Completed") ]]; then
+          # Add details about released bundle and indices
+          echo -e "\n## Release info\n" >> "$PR_NAME/comment.md"
+          RELEASE_INFO_DIR_PATH="$(workspaces.output.path)/release_info"
+          if [[ -d "$RELEASE_INFO_DIR_PATH" ]]; then
+            if [[ -f "$RELEASE_INFO_DIR_PATH/released_bundle.txt" ]]; then
+              echo -e "\n*Released bundle:* \n" >> "$PR_NAME/comment.md"
+              cat "$RELEASE_INFO_DIR_PATH/released_bundle.txt" >> "$PR_NAME/comment.md"
+            fi
+            if [[ -f "$RELEASE_INFO_DIR_PATH/updated_indices.txt" ]]; then
+              echo -e "\n*Updated indices:* \n" >> "$PR_NAME/comment.md"
+              cat "$RELEASE_INFO_DIR_PATH/updated_indices.txt" >> "$PR_NAME/comment.md"
+            fi
+          else
+            echo -e "\nNothing was released.\n" >> "$PR_NAME/comment.md"
+          fi
+
+          # Add info how to update catalog for FBC bundles
+          if [[ "$(params.fbc-enabled)" == "true" ]]; then
```

Comment:
This comment should be added only when FBC is enabled for the bundle and pipelines release an operator. If a FBC is enabled and the pipeline releases just indices the comment should be skipped.